api/src/main/resources/Java/libraries/okhttp-gson/api.mustache

```diff
@@ -0,0 +1,277 @@
+{{!
+
+This is an AoU-specific extension of the default Swagger-codegen okhttp API implementation class.
+
+This file was originally copied from the swagger-codegen repo in December 2019; the swagger-codegen file
+was last modified on Oct 2017: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/api.mustache
+
+AoU-specific additions have been clearly bookended with comments marked "Begin AoU-specific additions".
+
+}}
+{{>licenseInfo}}
+
+package {{package}};
+
+import {{invokerPackage}}.ApiCallback;
+import {{invokerPackage}}.ApiClient;
+import {{invokerPackage}}.ApiException;
+import {{invokerPackage}}.ApiResponse;
+import {{invokerPackage}}.Configuration;
+import {{invokerPackage}}.Pair;
+import {{invokerPackage}}.ProgressRequestBody;
+import {{invokerPackage}}.ProgressResponseBody;
+{{#performBeanValidation}}
+import {{invokerPackage}}.BeanValidationException;
+{{/performBeanValidation}}
+
+import com.google.gson.reflect.TypeToken;
+
+import java.io.IOException;
+
+{{#useBeanValidation}}
+import javax.validation.constraints.*;
+{{/useBeanValidation}}
+{{#performBeanValidation}}
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
+import javax.validation.executable.ExecutableValidator;
+import java.util.Set;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+{{/performBeanValidation}}
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+{{! Begin AoU-specific additions }}
```

Comment:
Can this not be an open/close pair like the others? E.g. `#allOfUsAddition`?

Reply:
Also, I'm wondering if putting the aou-specific code into separate files and including them into the main template would make sense. Then it would be more obvious what's ours and what's theres.

Reply:
Wow – that worked great. This helps mitigate the concern that upgrading to future Swagger versions would be difficult; the AoU-specific material is now much more isolated. 👍